tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Do note that, by default, ts_library and necessary rule use "//:tsconfig.json" as the default label for tsconfig field. https://github.com/bazelbuild/rules_nodejs/blob/432f88c8acc47b95ae6f368d3c5b7fe3e65baef7/packages/typescript/src/internal/build_defs.bzl#L424-L425